server/lib/tuist/runs.ex

```diff
@@ -34,6 +34,20 @@ defmodule Tuist.Runs do
     end
   end
 
+  def get_latest_test_by_build_run_id(build_run_id) do
+    query =
+      from(t in Test,
+        where: t.build_run_id == ^build_run_id,
+        order_by: [desc: t.ran_at],
+        limit: 1
+      )
```

Comment:
technically, a build can be associated with multiple tests. For now, we will show just the latest one, although we could certainly in the future show all of them.